src/uploader.js

```diff
@@ -151,6 +151,29 @@ define([
                 .addClass('fd-mm-preview-container'),
             ICONS = widget.mug.form.vellum.data.javaRosa.ICONS;
 
+        widget.getUploaderModal = function () {
+            return $("#" + SLUG_TO_UPLOADER_SLUG[widget.form]);
+        };
+
+        widget.updateModalExistingFile = function (objectMap, isNew) {
```

Comment:
It's not obvious to me what `isNew` means in this context. Seems like it causes an "upload completed" thing to become visible. Would it make sense to change the name to `isCompleted`?

Reply:
sure - c423dfd3